Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -3,11 +3,9 @@ Copyright (c) 2021 Bolton Bailey. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Bolton Bailey
 -/
-import Mathlib.Data.Nat.PrimeFin
 import Mathlib.Data.Nat.Totient
-import Mathlib.Data.Finset.LocallyFinite
-import Mathlib.Data.Nat.Count
 import Mathlib.Data.Nat.Nth
+import Mathlib.NumberTheory.SmoothNumbers
```

Comment:
This file is a leaf so it's probably fine, but I think these days we're hesitant about changing/adding imports, fwiw

Reply:
The alternatives would be
1. add `Mathlib.NumberTheory.PrimeCouting` as an import to the new file and put the lemma there. This import would be needed for this statement *only*, so I was hesitant, since this new file will not stay a leaf file.
2. have a separate file for this one lemma with the two relevant imports.

I'm fine with all possibilities. I just would like some consensus what the preferred solution should be.

Reply:
If no more comments are made, then I suggest to leave it as it is now.